src/parser.yy

```diff
@@ -626,10 +626,15 @@ sizeof_expr:
         |       SIZEOF "(" expr ")"                         { $$ = driver.ctx.make_node<ast::Sizeof>($3, @$); }
                 ;
 
+struct_field:
+                external_name                       { $$ = $1; }
+        |       struct_field DOT external_name      { $$ = $1 + "." + $3; }
```

Comment:
What about making this an array/vector of strings instead of a large concatenated string?

Reply:
Thanks, just update, please review.